be/src/vec/functions/function_timestamp.cpp

```diff
@@ -33,6 +33,16 @@ namespace doris::vectorized {
 struct StrToDate {
     static constexpr auto name = "str_to_date";
 
+    static bool is_variadic() { return false; }
+
+    static DataTypes get_variadic_argument_types() {
```

Comment:
Since the function with variadic arguments are less efficient when we search it, I think you should keep these functions with no variadic arguments

Reply:
done